scss/themes/_branded.scss

```diff
@@ -0,0 +1,67 @@
+@mixin oTopperThemeBranded {
+	.o-topper__content {
+		grid-row: 2 / span 3;
+		@include oGridRespondTo($until: L) {
+			padding: 20px 0;
+		}
+	}
+
+	.o-topper__read-next {
+		grid-row: 2 / span 3;
+	}
+
+	@include oGridRespondTo($until: L) {
+		//Make the background really full width
+		.o-topper__background {
+			position: absolute;
+		}
+	}
+}
+
+@mixin oTopperHasHeadshot {
+	.o-topper__tags,
+	.o-topper__standfirst {
+		box-sizing: border-box;
+		padding-right: 100px;
+
+		@include oGridRespondTo(M) {
+			padding-right: 150px;
+		}
+	}
+
+	.o-topper__headline {
+		@include oGridRespondTo(M) {
+			padding-right: 150px;
+		}
+	}
+}
+
+@mixin oTopperThemeOpinion {}
+
+@mixin oTopperOpinionGenre {
+	@extend %_o-topper__tag;
+	font-weight: normal;
+	color: oColorsGetPaletteColor('oxford');
+}
+
+@mixin oTopperHeadshot {
```

Comment:
As this and `oTopperHeadshotImage` are shared by multiple themes probably good these live outside the theme specific file. Maybe they belong in `scss/_elements.scss`?

Reply:
Same with `oTopperHasHeadshot`

Reply:
headshot is only currently used in the opinion/branded theme (opinion is a subtheme of branded)

Reply:
Does `.o-topper--opinion` only work when used with  `.o-topper--branded`?

Reply:
turns out yes, in that `.o-topper--opinion` doesn't actually have any styles, it's all done by `.o-topper--branded` and `.o-topper--color-sky` 😬

what do you think is best here? dropping `opinion` as a theme and just hanging off `branded`, `sky` and the `headshot` family of classes?

Reply:
Yeah definitely onboard with dropping `opinion` as a blank modifier. It looks like everything under `oTopperThemeElements` can actually be output within `oTopperThemeBranded` because they're all elements or variants of the branded theme? https://github.com/Financial-Times/o-topper/pull/1/files#diff-c42b41ef1770d78a14959fc99c1b148eR1

Reply:
i've thought about it and i'm going to keep the opinion theme around. while it's just the same as branded now, it's not unthinkable it will change, and having opinion as a semantic thing (rather than just "it's a blue branded topper") is a good thing to have imo

Reply:
totally vindicated now the opinion theme is setting tags font size 😉